instructure/canvas-ios
# Add lock explanations to modules

refs: MBL-14586
affects: student
release note: Added lock explanations to modules

test plan:
- narmstrong > student12 > CS 1400 > Modules
- Tap the lock icon for the "Midnight" module
- It should give an explanation about the prerequisite modules
- Tap the lock icon for the "Module 1" module
- It should give an explanation about the unlock date
